src/utils/embedding_utils.py

```
@@ -106,6 +178,13 @@ async def create_embedding(text: str) -> List[float]:
         print(f"Error creating embedding: {e}")
         return [0.0] * EMBEDDING_DIMENSION
 
+async def close_http_client():
```

Ruff (0.12.2)

Missing return type annotation for public function `close_http_client`

Add return type annotation: `None`

(ANN201)